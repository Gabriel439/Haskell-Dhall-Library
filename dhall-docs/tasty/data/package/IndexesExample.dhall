{- Checking indexes -}

let x : Bool = True
let x : Natural = 1

in x@1
