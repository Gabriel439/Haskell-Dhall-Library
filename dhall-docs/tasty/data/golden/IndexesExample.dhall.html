<!DOCTYPE HTML>
<html
  ><head
    ><title
    >/IndexesExample.dhall</title
    ><link href="index.css" type="text/css" rel="stylesheet"
    /><link href="https://fonts.googleapis.com/css2?family=Fira+Code:wght@400;500;600;700&amp;family=Lato:ital,wght@0,400;0,700;1,400&amp;display=swap"
            type="text/css" rel="stylesheet"
    /><script src="index.js" type="text/javascript"
    /><meta charset="UTF-8"/></head
  ><body
    ><div class="nav-bar"
      ><img src="dhall-icon.svg" class="dhall-icon"
      /><p class="package-title"
      >test-package</p
      ><div class="nav-bar-content-divider"
      /><a id="switch-light-dark-mode" class="nav-option"
      >Switch Light/Dark Mode</a></div
    ><div class="main-container"
      ><h2 class="doc-title"
        ><span class="crumb-divider"
        >/</span
        ><a href="index.html"
        >test-package</a
        ><span class="crumb-divider"
        >/</span
        ><span href="index.html" class="title-crumb"
        >IndexesExample.dhall</span></h2
      ><a data-path="/IndexesExample.dhall" class="copy-to-clipboard"
        ><i
          ><small
          >Copy path to clipboard</small></i></a
      ><br
      /><div class="doc-contents"
        ><p
          >This example shows that <code
          >dhall-docs</code> will find render the correct type on the
index respecting variable indices</p>
</div
      ><h3
      >Source</h3
      ><div class="source-code"
        ><pre
          ><span class="dhall-keyword"
          >let</span
          > <span class="dhall-label"
          >x</span
          ><br
          />    <span class="dhall-syntax"
          >:</span
          > <span class="dhall-builtin"
          >Bool</span
          ><br
          />    <span class="dhall-syntax"
          >=</span
          > <span class="dhall-builtin"
          >True</span
          ><br
          /><br
          /><span class="dhall-keyword"
          >let</span
          > <span class="dhall-label"
          >x</span
          ><br
          />    <span class="dhall-syntax"
          >:</span
          > <span class="dhall-builtin"
          >Natural</span
          ><br
          />    <span class="dhall-syntax"
          >=</span
          > <span class="dhall-literal"
          >1</span
          ><br
          /><br
          /><span class="dhall-keyword"
          >in</span
          >  <span class="dhall-label"
          >x@1</span></pre></div></div></body></html>